changelog.txt

```diff
@@ -1,3 +1,8 @@
+4.7.0
+============
+New Features:
+**Added Subject Name + Issuer authentication to the acquire token by authorization code and acquire token by refresh token flows with the WithSendX5C() api on the confidential client application.** All confidential client authentication flows will not have acces to this feature. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1490)
```

Comment:
"will now have access" ? or not have access? also add another "s" to access :) thx.  looks cool

Reply:
should be "will now have access"

Reply:
okay..because it says "will not"